packages/macos-at-driver-server/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bocoup/macos-at-driver-server",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Although we intend to release your fix immediately in this case, I'd still prefer to do this with a dedicated commit.

```suggestion
  "version": "0.0.7",
```